src/pause-play/js/main.js

```diff
@@ -66,18 +79,21 @@ class PeerConnection {
       this.remoteView.srcObject = e.stream;
     };
 
-    var onCreateOfferSuccess = this.onCreateOfferSuccess.bind(this);
     this.localConnection
       .createOffer({offerToReceiveAudio: 1, offerToReceiveVideo: 1})
-      .then(onCreateOfferSuccess, logError);
+      .then((desc) => {
+        this.onCreateOfferSuccess(desc);
+      }, logError);
   }
 
   onCreateOfferSuccess(desc) {
     this.localConnection.setLocalDescription(desc);
     this.remoteConnection.setRemoteDescription(desc);
 
-    var onCreateAnswerSuccess = this.onCreateAnswerSuccess.bind(this);
-    this.remoteConnection.createAnswer().then(onCreateAnswerSuccess, logError);
+    this.remoteConnection.createAnswer().then(
+      () => {
```

Comment:
Ok I see, this could be that it does not lint properly for es6 as well. Lets leave it as is for now.